AsyncDisplayKit/ASCellNode.m

```diff
@@ -134,6 +134,12 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event
   [(_ASDisplayView *)self.view __forwardTouchesCancelled:touches withEvent:event];
 }
 
+- (void)updateScrollSituationWithScrollVIew:(UIScrollView *)scrollView
+{
+    // TODO(Max): Fix the cellFrame here
+    [self.layoutDelegate scrollViewDidScroll:scrollView cellFrameInScrollView:CGRectZero];
```

Comment:
We should implement this in hey way that works even if the developer overrides this method and forgets to call the super method. You can do this by using a different method name, such as one that starts with _, as the core internal implementation that is responsible for calling both the method that might be subclassed as well as the delegate.

I think we should replace the property value added to control whether we track visibility, And instead look at response to selector were both the delegate and the cell subclass